From 05f033868f7e8fdece13fa2861a5bda1f433ed23 Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Sun, 7 Feb 2021 08:45:33 -0800
Subject: [PATCH] Update env for SPIflash Nanos for R32.5.0 with Mender

Need to move the environment area and reduce its
size in order to fit redundant copies between RP4
and VER_b.

Signed-off-by: Matt Madison <matt@madison.systems>
---
 configs/p3450-0000_defconfig | 6 +++---
 configs/p3541-0000_defconfig | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/configs/p3450-0000_defconfig b/configs/p3450-0000_defconfig
index 0fe9b5fb52..9907eaedb8 100644
--- a/configs/p3450-0000_defconfig
+++ b/configs/p3450-0000_defconfig
@@ -60,9 +60,9 @@ CONFIG_USB_XHCI_TEGRA=y
 CONFIG_USB_STORAGE=y
 CONFIG_DOS_PARTITION=y
 CONFIG_CMD_CACHE=y
-CONFIG_ENV_SIZE=0x10000
-CONFIG_ENV_OFFSET=0x3b0000
-CONFIG_ENV_OFFSET_REDUND=0x3c0000
+CONFIG_ENV_SIZE=0x8000
+CONFIG_ENV_OFFSET=0x3d0000
+CONFIG_ENV_OFFSET_REDUND=0x3d8000
 CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
 # CONFIG_ENV_IS_NOWHERE is not set
 # CONFIG_ENV_IS_IN_EEPROM is not set
diff --git a/configs/p3541-0000_defconfig b/configs/p3541-0000_defconfig
index 708e6d0398..4443cd9a7f 100644
--- a/configs/p3541-0000_defconfig
+++ b/configs/p3541-0000_defconfig
@@ -1,9 +1,9 @@
 CONFIG_ARM=y
 CONFIG_ARCH_TEGRA=y
 CONFIG_SYS_TEXT_BASE=0x80080000
-CONFIG_ENV_SIZE=0x10000
-CONFIG_ENV_OFFSET=0x3b0000
-CONFIG_ENV_OFFSET_REDUND=0x3c0000
+CONFIG_ENV_SIZE=0x8000
+CONFIG_ENV_OFFSET=0x3d0000
+CONFIG_ENV_OFFSET_REDUND=0x3d8000
 CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
 CONFIG_ENV_SECT_SIZE=0x1000
 CONFIG_NR_DRAM_BANKS=16
-- 
2.27.0

